lib/pharos/scripts/configure-kubelet-proxy.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -eu
+
+if [ ! -e /usr/bin/kubelet ]; then
+    mkdir -p /etc/systemd/system/kubelet.service.d
+    cat <<EOF >/etc/systemd/system/kubelet.service.d/5-pharos-kubelet-proxy.conf
```

Comment:
The condition here isn't ideal... `pharos-cluster up` gets stuck in the `Waiting etcd to launch ...` / `Waiting kubelet-proxy to launch ...` if you already have the kubelet installed.